rmf_demos_tasks/rmf_demos_tasks/dynamic_event_cancel_task.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python3
+
+# Copyright 2025 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import argparse
+import sys
+
+import rclpy
+
+from rclpy.action import ActionClient
+from rclpy.node import Node
+
+from rmf_task_msgs.action import DynamicEvent
+
+
+class DynamicEventActionClient(Node):
+
+    def __init__(self, argv=sys.argv):
+        super().__init__('simple_action_client')
+        parser = argparse.ArgumentParser()
+        parser.add_argument(
+            '-F',
+            '--fleet',
+            required=True,
+            type=str,
+            help='Fleet name'
+        )
+        parser.add_argument(
+            '-R',
+            '--robot',
+            required=True,
+            type=str,
+            help='Robot name'
+        )
+
+
+        parser.add_argument(
+            '-id',
+            '--id',
```

Comment:
```suggestion
            'id'
```

I suggest the above change to turn this into a positional argument. That way users don't need to choose between `--id` or `-id`.